lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```
@@ -1949,6 +1949,10 @@ definitions:
       nat_enabled:
         type: boolean
         example: true
+      enable_converged_core:
+        type: boolean
+        example: false
+        description: enable_converged_core is to Enable or Disable 5G SA.
```

```suggestion
        description: True if 5G SA should be enabled
```
What is SA btw? (Might want to expand the acronym in the description for noobs like me :) )

---

Yes I will update that